### PR TITLE
feat(crds): add dapr configuration yaml fields

### PR DESCRIPTION
**Pull request by 1046102779**
add httpPipeline version field and tracing stdout field.

### CODE REVIEW

**Comment by artursouza** [comment]
How do you plan to use these? Is there another PR for that?

File: charts/dapr/crds/configuration.yaml [hunk 1]
```diff
@@ -99,6 +99,8 @@ spec:
                           type: object
                         type:
                           type: string
+                        version:
```
Comment (artursouza) [inline charts/dapr/crds/configuration.yaml:102]: How would these new attributes be used?

**Review by artursouza — CHANGES_REQUESTED** [review]

File: charts/dapr/crds/configuration.yaml (hunk 1 above)
Comment (1046102779) [reply charts/dapr/crds/configuration.yaml:102]: yes.  version field is used in httpPipeline.

```golang
// github.com/dapr/dapr/pkg/components/middleware/http/registry.go
func (p *httpMiddlewareRegistry) getMiddleware(name, version string) (func(middleware.Metadata) http_middleware.Middleware, bool) {
      nameLower := strings.ToLower(name)
      versionLower := strings.ToLower(version)
      middlewareFn, ok := p.middleware[nameLower+"/"+versionLower]
      if ok {
          return middlewareFn, true
      }
      if components.IsInitialVersion(versionLower) {
          middlewareFn, ok = p.middleware[nameLower]
      }
      return middlewareFn, ok
  }
```

Comment (yaron2) [reply charts/dapr/crds/configuration.yaml:102]: This method is kicked off for `kind: Middleware` CRDs, not for `kind: Configuration`.

Middlewares are defined as a pipeline in a Configuration object, but the Configuration object itself does not need a version field.

As such, I advise to close this PR. (but thanks for opening it nonetheless!)

**Pull request closed without merging**